ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/StatementRewriter.java

```diff
@@ -131,67 +134,63 @@ protected Node visitArithmeticBinary(
       final ArithmeticBinaryExpression node,
       final Object context
   ) {
-    return node.getLocation()
-        .map(location ->
-            new ArithmeticBinaryExpression(location, node.getType(),
-                (Expression) process(node.getLeft(), context),
-                (Expression) process(node.getRight(), context)))
-        .orElse(new ArithmeticBinaryExpression(node.getType(),
-            (Expression) process(node.getLeft(), context),
-            (Expression) process(node.getRight(), context)));
-
+    // Don't change this if/else block so only one object gets instanciated.
```

Comment:
- instaciated -> instantiated
- maybe rephrase to something like "use an if/else block here (instead of isPresent.map(...).orElse(...)) so only one object gets instantiated (issue #1784)"